lib/sphinx/integration/extensions/thinking_sphinx/configuration.rb

```diff
@@ -144,11 +159,14 @@ def enforce_common_attribute_types_with_rt
 
   private
 
-  def build_mysql_client(privileged: false)
+  def build_mysql_client(privileged: false, host: nil)
     port = configuration.searchd.mysql41.presence
     port = DEFAULT_MYSQL_PORT if port.nil? || port.is_a?(TrueClass)
     vip_port = configuration.searchd.mysql41_vip.presence || port
 
-    ::Sphinx::Integration::Mysql::Client.new(shuffled_addresses, privileged ? vip_port : port)
+    ::Sphinx::Integration::Mysql::Client.new(
+      host ? [host] : shuffled_addresses,
+      privileged ? vip_port : port,
```

Comment:
Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.